must-gather/gather

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
+
+# copyright 2023.
+#
+# licensed under the apache license, version 2.0 (the "license");
+# you may not use this file except in compliance with the license.
+# you may obtain a copy of the license at
+#
+#     http://www.apache.org/licenses/license-2.0
+#
+# unless required by applicable law or agreed to in writing, software
+# distributed under the license is distributed on an "as is" basis,
+# without warranties or conditions of any kind, either express or implied.
+# see the license for the specific language governing permissions and
+# limitations under the license.
+#
+
+set -eu -o pipefail
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+source "${SCRIPT_DIR}"/common
+
+print_usage() {
+    echo -e "Usage: /usr/bin/gather <gather-collection-path>"
+}
+
+get_kepler_instance() {
+  log "getting kepler instance"
+  run oc get keplers.kepler.system.sustainable.computing.io kepler -oyaml "$BASE_COLLECTION_PATH/kepler.yaml"
```

Comment:
In case of Kepler instance is not created the execution gets terminated and the rest of the steps are skipped. I also believe we should add `|| true` at every step